app/javascript/gobierto_dashboards/webapp/containers/tender/TendersShow.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div>
+    <h1>{{ formattedTender.title }}</h1>
+
+    <p v-if="formattedTender.description">
+      {{ formattedTender.description }}
+    </p>
+
+    <div class="pure-g p_2 bg-gray">
+      <div class="pure-u-1 pure-u-lg-1-2">
+      </div>
+
+      <div class="pure-u-1 pure-u-lg-1-2">
+        <table>
+          <tr>
+            <th class="left">{{ labelTenderAmount }}</th>
+            <td>{{ formattedTender.initial_amount }}</td>
+          </tr>
+          <tr>
+            <th class="left">{{ labelStatus }}</th>
+            <td>{{ formattedTender.status }}</td>
+          </tr>
+          <tr>
+            <th class="left">{{ labelProcessType }}</th>
+            <td>{{ formattedTender.process_type }}</td>
+          </tr>
+        </table>
+      </div>
+    </div>
+
+  </div>
+</template>
+
+<script>
+import { formatCurrency } from "../../lib/utils.js";
+
+export default {
+  name: 'TendersShow',
+  data() {
+    return {
+      tendersData: this.$root.$data.tendersData,
+      tender: null,
+      formattedTender: null,
+      labelTenderAmount: I18n.t('gobierto_dashboards.dashboards.contracts.tender_amount'),
```

Comment:
There's a filter also for translations, called `translate`, in `app/javascript/lib/shared/modules/vue-filters.js`

Instead of declaring data variables for translations I'd prefer to use it in the template like this:

{{ 'gobierto_dashboards.dashboards.contracts.tender_amount' | translate }}

Reply:
That does not seems to work, though. I think `translate` is supposed to receive an Object and not a string?

Reply:
Hmmm, correct. I want to create at some point a filter to translate strings in the template and avoid so many variables in data (it happens too in Gobierto Data). We'll addresss this in the future.